.eleventy.js

```diff
@@ -80,6 +80,15 @@ module.exports = function(config) {
     return content
   })
 
+  // Правит пути к демкам внутри советов
+  config.addTransform('fix-demos', (content, outputPath) => {
```

Comment:
```suggestion
  config.addTransform('fixDemos', (content, outputPath) => {
```
У нас другой кейс внутри JS, был неправ